app/components/status/error_page_component.html.erb

```diff
@@ -5,5 +5,8 @@
   <div class="ml-4 fw-600">
     <h2 class="h3"><%= header %> (<%= code %>)</h2>
     <%= message %>
+    <p>If you believe this is an error, please report it via the 
```

Comment:
@dbranchini : is the language here a little weird? It's definitely an error, so I'm not sure what we're asking the user to do